bigquery_etl/view/generate_stable_views.py

```diff
@@ -166,16 +192,23 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, schema: SchemaF
             full_view_id=full_view_id,
         )
     )
-    with tempfile.TemporaryDirectory() as tdir:
-        tfile = Path(tdir) / "view.sql"
-        with tfile.open("w") as f:
-            f.write(full_sql)
-        dryrun = DryRun(str(tfile))
-        if 404 in [e.get("code") for e in dryrun.errors()]:
-            print(
-                f"Not creating {schema.user_facing_view} since stable table does not exist"
-            )
-            return
+    stable_table_missing = False
+    if stable_tables is not None:
```

Comment:
using `if stable_tables:` would results in falling back to dry-run when no stable tables are detected, which occurs when the active credentials don't have permissions to list datasets and tables in `moz-fx-data-shared-prod`